tools/debian/control

```diff
@@ -0,0 +1,116 @@
+Source: cockpit
+Section: admin
+Priority: optional
+Maintainer: Michael Biebl <biebl@debian.org>
```

Comment:
I just filled something in there, because that field must not be empty for a successful build.
I would prefer if someone else claimed maintainership, just to avoid confusion.
That said, I wouldn't mind too much, if you just keep me there for now